src/electronic-ids/pcsc/LatEIDIDEMIAv2.cpp

```diff
@@ -32,10 +32,20 @@
 using namespace pcsc_cpp;
 using namespace electronic_id;
 
+// Workaround gcc 14.2 bug
+struct VectorComparator
```

Comment:
Put it in a common header and reuse instead of duplicating in two .cpp files.

Add a comment with the upstream GCC bug ID so we can track when it can be removed.